.cirrus.yml

```diff
@@ -378,7 +378,7 @@ task:
     # we don't pass --keep-going to the build: failures can be debugged with
     # logs of other jobs.
     after_cache_script: *after_cache_script
-    build_script: make -j${JOBS} test/test
+    build_script: make -j${JOBS} test
```

Comment:
This builds the binaries, but doesn't run them. Please change the `test_script` too (just `make ci-check` there, perhaps? I don't recall why the current `test_script` explicitly runs just the C++ suite.)

Ditto for UBSan.

Reply:
> This builds the binaries, but doesn't run them

It looks like that works as intended (as written by you in https://github.com/newsboat/newsboat/pull/990).
This line makes sure that the test binary and the "http test server" binary are both build.
The next line will make sure the tests are run (the `test/test` binary will automatically start the http test server)

> I don't recall why the current test_script explicitly runs just the C++ suite.

I expect that is answered by the comment above (added in https://github.com/newsboat/newsboat/pull/990):
```
    # Sanitizers only apply to C++, so we only build and run C++ tests. Also,
    # we don't pass --keep-going to the build: failures can be debugged with
    # logs of other jobs.
```